### ISSUE TITLE
What happend to the ISerializerFactory which was there to help the NewtonsoftJson in the Renderer

### ISSUE THREAD

**Issue by VinayShiva** [BrightOpen/log4net.Ext.Json#2]
This seems to be an issue when the minor version is upgraded its not having the Factory 

```
   <renderer type="log4net.ObjectRenderer.JsonObjectRenderer, log4net.Ext.Json">
        <factory type="log4net.Ext.Json.Serializers.Newtonsoft.NewtonsoftFactory,
                     log4net.Ext.Json.Serializers.Newtonsoft" />
```

This is not working.

**Comment by jocutajar** [comment]
Hi @VinayShiva , I haven't worked on this project in a good while. I would take a pull request if you feel like. Kind regards, Rob

**Comment by jocutajar** [comment]
Hi, I've released 2.0.8.2 with a companion of log4net.Ext.Json.Net that provides the Newtonsoft.Json powered renderer. Would that work for you? Kind regards, Rob

**Comment by jocutajar** [comment]
New release adresses newtonsoft.json
